mtags/src/main/scala-3.0.0-M3/scala/meta/internal/mtags/VersionSpecificEnrichments.scala

```diff
@@ -0,0 +1,23 @@
+package scala.meta.internal.mtags
+
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.Types.Type
+import dotty.tools.dotc.core.Flags.EmptyFlags
+import dotty.tools.dotc.util.NoSourcePosition
+
+trait VersionSpecificEnrichments {
+
+  extension(context: Context) {
+    def findRef(name: Name): Type = {
+      context.typer.findRef(
+        name,
+        defn(using context).AnyType,
+        EmptyFlags,
+        EmptyFlags,
```

Comment:
There is one extra parameter here, which is missing in M1 and M2

Reply:
Oops, didn't realize it, thanks for fixing it 👍 
it looks good! (and good to see the use of the `extension` keyword!)